inlineplz/linters/config/.coafile

```diff
@@ -0,0 +1,3 @@
+[all]
+bears = LineLengthBear
+files = *
```

Comment:
does this mean it'll enforce line length on all files?

Reply:
Yes, if the project does not specify a dotfile. Basically coala blows up if `bears` and `files` are not specified, so I wanted to put something in there. I've already omitted coala from autorun, perhaps we want to leave it disabled as well?